Godeps/Godeps.json

```diff
@@ -904,10 +880,6 @@
 		{
 			"ImportPath": "k8s.io/utils/clock",
 			"Rev": "aedf551cdb8b0119df3a19c65fde413a13b34997"
-		},
-		{
-			"ImportPath": "k8s.io/utils/clock/testing",
```

Comment:
looks like this is removed from the godeps, but the packages in `vendor/` are still there.  I think some of the other packages are in the same boat